frontend/packages/rhoas-plugin/src/components/service-table/ServiceInstanceTable.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Bullseye,
+  Title,
+  EmptyStateIcon,
+} from '@patternfly/react-core';
+import {
+  sortable,
+  Table,
+  TableHeader,
+  TableBody,
+  RowSelectVariant,
+  SortByDirection,
+  Tbody,
+  Tr,
+  Td,
+} from '@patternfly/react-table';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+import { Timestamp } from '@console/internal/components/utils';
+import './ServiceInstanceTable.css';
```

Comment:
nit:  sort the import. Style should import at last.  and can we use `scss` instead of css?

Reply:
Ah.. lesson learned. Lint issues rearranged that for me. I need to separate those if I need them on the bottom